### DISCUSSION TITLE
Feature request: Support VLANs as target LAN upon configuring

### DISCUSSION THREAD

**Discussion by jilderda** [fabianishere/udm-iptv#304] [Ideas]
When having your local network segmented into VLANs, it would be good if 'udm-iptv configure' shows those as an option for the LAN.  For instance, I have udm-iptv deliver the stream to eth0.50.  That is, I have the line "LAN Interfaces: eth0.50" in /etc.udm-iptv.conf.

**Comment by dmutsaers** [comment]
See "Add support for UXG Lite LAN interfaces"
https://github.com/fabianishere/udm-iptv/pull/291
